server/routerlicious/packages/services-core/src/runWithRetry.ts

```diff
@@ -26,14 +27,17 @@ export async function runWithRetry<T>(
             logger?.info(`Error running ${callName}: retryCount ${retryCount}, error ${error}`);
             if (shouldIgnoreError !== undefined && shouldIgnoreError(error) === true) {
                 logger?.info(`Should ignore error for ${callName}`);
+                Lumberjack.info(`Should ignore error for ${callName}`);
                 break;
             } else if (shouldRetry !== undefined && shouldRetry(error) === false)
             {
                 logger?.info(`Should not retry ${callName}`);
+                Lumberjack.info(`Should not retry ${callName}`);
```

Comment:
Should we also use `Lumberjack.error()` here? also, should we explicitly say "rejecting" at the end of the error message to make it clear the retry will not be executed and the call will fail?

Reply:
Another thing that could be helpful is saying `should not retry ${callName} for the current error.`. And then, pass the error object to `Lumberjack.error()`